updated_permissions/vm-protection/permission-set-basic.json

```diff
@@ -0,0 +1,228 @@
+[
+  {
+    "Actions": [
+      {
```

Comment:
Can we rename this to permission-group-basic.json
We use permission group term everywhere and it would be good to stick to a single terminology. Can you also rename other file accordingly.